pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
+name = "MoUSE"
+version = "1.0.0"
+description = "Joseph the MoUSE: Application for rodent ultrasonic identification - backend. Described in: Kania, Adam, et al. \"Joseph the MoUSE—Mouse Ultrasonic Sound Explorer.\" SoftwareX 25 (2024): 101606."
+authors = ["See autors.md file in the repo"]
```

Comment:
Lets keep the mailing list here.
https://python-poetry.org/docs/pyproject#authors

![image](https://github.com/user-attachments/assets/235cd343-90ce-4e04-844c-c0b9ab8d9173)